shared/protocol/src/main/java/io/pravega/shared/protocol/netty/CommandEncoder.java

```diff
@@ -181,7 +244,7 @@ private void breakFromAppend(ByteBuf data, ByteBuf pendingData, ByteBuf out) {
     }
 
     @SneakyThrows(IOException.class)
-    private void writeMessage(AppendBlock block, ByteBuf out) {
+    private void writeMessage(AppendBlock block, int blkSize, ByteBuf out) {
```

Comment:
blkSize???

Reply:
this only for AppendBlock; where approximate the number of data bytes that we determine while writing appendBlock

Reply:
I am referring to the name of the argument.